src/styles/index.scss

```diff
@@ -1,3 +1,169 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  border: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+
+  color: #616070;
```

Comment:
- SCSS Variables are used for main values and placed in a separate file

Reply:
ok